filters/britishness/filter.py

```diff
@@ -0,0 +1,149 @@
+import json
+import re
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from initialize import spacy_nlp
+import spacy
+
+
+class BritishnessFilter(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self, britwords_required=1):
+        super().__init__()
+        self.britwords_required = britwords_required
+        # self.keywords = keywords
+        # self.nlp = spacy_nlp if spacy_nlp else spacy.load("en_core_web_sm")
+
+    def filter(self, sentence: str = None) -> bool:
+        britwords = get_britwords()
+
+        num_britwords = 0
+        for bword in britwords:
+            num_britwords += len(re.findall("(?<![a-zA-Z])"+bword+"(?![a-zA-Z])", sentence))
+            num_britwords += len(re.findall("(?<![a-zA-Z])"+bword[0].upper()+bword[1:]+"(?![a-zA-Z])", sentence))
+
+        return (num_britwords >= self.britwords_required)
+
+def get_britwords():
+    britwords = []
+
+    spelling_json_file = open('filters/britishness/spelling_map.json')
+    spelling_map = json.load(spelling_json_file)
+
+    britwords += spelling_map.values()
+
+    vocab_map = {'Scotch tape': 'sellotape',
```

Comment:
For cleaner code, better to move this map also in a JSON file, something like below:

{
  "vocab": {
    'Scotch tape': 'sellotape'
  },
  "spelling": {
    "accessorize": "accessorise"
  }
}